charts/mailpit/templates/deployment.yaml

```
@@ -32,6 +32,7 @@ spec:
       {{- end }}
     spec:
       automountServiceAccountToken: false
+      {{- include "mailpit.renderPullSecrets" ( dict "imageRoot" .Values.image "context" $) | nindent 6 }}
```

dependency bitnami/common already defines renderPullSecrets :)

```suggestion
      {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.image) "context" $) | nindent 6 }}
```

---

Well that simplifies things 👍🏼 thanks!